### PR TITLE
Benefits of solar pv installation

### PR DESCRIPTION
**Pull request by ldodds**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -2295,4 +2295,59 @@
             ORDER BY alert_generation_runs.school_id, alert_generation_runs.created_at DESC) latest_runs
     WHERE (data.alert_generation_run_id = latest_runs.id);
   SQL
+  create_view "solar_generation_summaries", sql_definition: <<-SQL
```
Comment (urbanwide) [inline db/schema.rb:2298]: I think this could be from another PR?

**Review by urbanwide — APPROVED** [review]
Looks good - just one comment about schema.rb

File: db/schema.rb (hunk 1 above)
Comment (ldodds) [reply db/schema.rb:2298]: Yep, forgot to rollback when jumping between PRs
